src/amo/components/Footer.js

```diff
@@ -37,7 +37,7 @@ export class FooterBase extends React.Component {
             </a>
           </li>
           <li>
-            <a href={`https://developer.mozilla.org/${lang}/Add-ons/AMO/Policy/Contact`}
+            <a href={`https://developer.mozilla.org/en-US/Add-ons/AMO/Policy/Contact`}
```

Comment:
I think it would be better to omit the locale entirely; then MDN will at least try to redirect to the client's `Accept-Language` locale.